src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -167,4 +236,20 @@ export class ProjectsSummaryController implements angular.IController {
     $('.catalog-projects-summary-modal-backrop').remove();
   }
 
+  private getRecentlyViewedItems() {
+    // recentItems is an array of uids
+    let recentItems: any = this.RecentlyViewed.getItems();
+
+    // replace uids with IServiceItems
+    let items: any = _.map(recentItems, (uid: any) => {
+      return this.allItems[uid];
```

Comment:
This method only gets called after serviceClasses and imageStreams have been passed in.